src/javascript/JContent/actions/expandTree.jsx

```diff
@@ -28,7 +29,8 @@ const GetAncestorsQuery = gql`
 export const expandTree = (path, client) => {
     return client.query({query: GetAncestorsQuery, variables: {path}}).then(res => {
         let node = res.data.jcr.nodeByPath;
-        const acc = registry.find({type: 'accordionItem', target: 'jcontent'}).find(acc => acc.canDisplayItem(node));
+        const params = getCanDisplayItemParams(node);
```

Comment:
so in that case params is always `= {selectionNode:node}`
```suggestion
        const params = {selectionNode: node};
```

Reply:
This and regex modification brings me back to the issue of locating "files" folder because it is now trying to match selection node with ".../files/.*" while selection node is actually a folder with path ending at ".../files". We have the same issue with ".../contents" folder. That's why I added that type check thinking that it was actually dependent on node type.